lib/new_relic/agent/distributed_tracing/distributed_trace_payload.rb

```diff
@@ -101,10 +101,6 @@ def current_segment_id(transaction)
             transaction.current_segment.guid
           end
         end
-
-        def connected?
```

Comment:
Was removing this method and instead calling `Agent.instance.connected?` a cleanup, or related to lambda code?

Reply:
A little bit of both. For serverless/lambda's sake I originally introduced a new `ready?` method that read better than `connected?` in a serverless context. Later I took the agent's serverless mode even more minimalistic and stopped using the worker thread entirely, so there was no need to perform a `ready?` check anymore. When reverting out the `ready?` work, I decide to leave these refactors because I love being able to search for and find only a single `def connected?` line instead of the number we had before. So it started out lambda specific and now it's just a cleanup.

Reply:
Thank you for the explanation! ◡̈